site/en/docs/crux/dashboard/index.md

```diff
@@ -41,11 +41,68 @@ The CrUX Dashboard is a [Looker Studio](https://cloud.google.com/looker-studio)
 
 ## Accessing the CrUX Dashboard
 
-To use the existing CrUX Dashboard you need to [lauch the dashboard, passing the origin as a query param](https://datastudio.google.com/u/0/reporting/bbc5698d-57bb-4969-9e07-68810b9fa348/page/keDQB?params=%7B%22origin%22:%22wwww.example.com%22%7D). To make this easier–since the origin needs to be URL encoded—you can use [Rick Viscomi](https://twitter.com/rick_viscomi)'s [CrUX Dash Launcher](https://rviscomi.github.io/crux-dash-launcher/) which takes the URL as an input and constructs the dashboard URL.
+To use the existing CrUX Dashboard you need to lauch the dashboard, passing the origin as a query param. To make this easier–since the origin needs to be URL encoded—you can use the below launcher, which takes an origin or URL as an input and launches the appropriate dashboard URL for that origin:
+
+<style>
+  form {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    max-width: 800px;
+  }
+  #origin {
+    width: 100%;
+    font-size: 2em;
+    line-height: 1.3em;
+    text-align: center;
+  }
+  #origin[aria-invalid=true] {
+    outline: red dashed 2px;
+  }
+  #origin-submit {
+    line-height: 2.25em;
+  }
+</style>
+
+<form id="form">
+  <input id="origin" aria-label="Origin" type="text" placeholder="https://developer.chrome.com" required>
+  <button id="origin-submit" class="bg-primary button-filled color-bg material-button">Go</button>
+</form>
+
+*Note: if the protocol (`http` or `https`) is not provided, then `https` will be assumed. Paths will be stripped as the CrUX Dashboard only provides data at an origins level.*
+
+<script>
+var form = document.getElementById('form');
+var origin = document.getElementById('origin');
+form.addEventListener('submit', function(e) {
+  try {
+    origin.removeAttribute('aria-invalid');
+    e.preventDefault();
+    var url = origin.value.trim();
+    if (!url.startsWith('http')) url = 'https://' + url;
+    url = new URL(url);
+    var encoded_origin = encodeURIComponent(url.origin);
+    var url = 'https://datastudio.google.com/reporting/bbc5698d-57bb-4969-9e07-68810b9fa348/page/keDQB?params=%7B%22origin%22:%22' + encoded_origin + '%22%7D';
+    window.location = url;
+  } catch {
```

Comment:
Will this actually catch invalid inputs? If I enter `test` and click submit, it redirects me to a broken dashboard. The existing launcher used `input[type=url]` to catch these kinds of errors natively, WDYT about using that here?

Reply:
I wanted the ability to not have to specify scheme for simplicity. So you can enter `web.dev` and it works for example. `type=url` is a bit more restrictive for things like that.

The existing launcher does catch `test` but does not catch `http://test` as that is a valid URL (though not a valid external one - `http://localhost` is an example of a one word valid url) and I'm now deliberately added a default `https://` scheme if one is not provided.

URL parsing [is apparently complex](https://stackoverflow.com/questions/5717093/check-if-a-javascript-string-is-a-url) so `new URL()` is the best way of checking this IMHO but does allow single word domains.

So I added an extra check to ensure there's at least one dot in there (and not at beginning or end) in https://github.com/GoogleChrome/developer.chrome.com/pull/5732/commits/dc91e6354be4d673055f251b5d9421e1c609e5ca